source/version.js

```diff
@@ -55,3 +55,24 @@ module.exports.validate = version => {
 		throw new Error('Version should be a valid semver version.');
 	}
 };
+
+module.exports.satisfiesRequiredVersion = async function (pkgName, installedVersion) {
```

Comment:
I think that this method should be moved to `source/prerequisite-tasks.js`, since it's out of the scope of this file.